picopass/scenes/picopass_scene_config.h

```diff
@@ -15,7 +15,7 @@ ADD_SCENE(picopass, write_card_failure, WriteCardFailure)
 ADD_SCENE(picopass, read_factory_success, ReadFactorySuccess)
 ADD_SCENE(picopass, write_key, WriteKey)
 ADD_SCENE(picopass, key_menu, KeyMenu)
-ADD_SCENE(picopass, elite_dict_attack, EliteDictAttack)
 ADD_SCENE(picopass, emulate, Emulate)
 ADD_SCENE(picopass, loclass, Loclass)
 ADD_SCENE(picopass, key_input, KeyInput)
+ADD_SCENE(picopass, dict_attack, DictAttack)
```

Comment:
Some of these seem like changes that at independent of the NFC refactor.  If that is the case, could they be pulled out and done as an independent PR(s) so that it is easier to see the changed the refactor requires?

Reply:
+1 on this. It seems like it's a NFC refactor, plus major cleanup at the same time.

I'm also concerned that some of the decisions, especially around emulation splitting to looping over an array of handlers rather than a switch statement, will affect emulation speed breaking compatibility with some readers. There's definitely some code that doesn't look the clanest but was done that way for performance reasons to make it work. Rather difficult determine what has broken something when there's so many things done together.

Reply:
@nvx sorry for late response, lost your message.

I think that introduced array of handlers in emulation part doesn't affect flipper response time much but gives much cleaner code. Meanwhile we improved ISO15693 signal decoder and encoder in NFC library, which helped us to achieve better timing. Still we need to test it, and if it doesn't work good, we can apply some optimizations.